<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroPage</class>
 <widget class="QWidget" name="IntroPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="remasterResources.qrc">:/resources/Вставленное изображение.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_ageGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Выберите возрастную группу</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_ageGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>Возрастная группа</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_sex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Выберите пол</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_m">
     <property name="text">
      <string>Мужской</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rb_w">
     <property name="text">
      <string>Женский</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="l_copetitionStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Выберите статус соревнований</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lw_copetitionStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="remasterResources.qrc"/>
 </resources>
 <connections/>
</ui>
